apps/api/src/auth/dto/recovery-password.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { ApiProperty } from '@nestjs/swagger'
+import { Expose } from 'class-transformer'
+import { IsJWT, IsNotEmpty, IsString } from 'class-validator'
+
+export class RecoveryPasswordDto {
+  @ApiProperty()
+  @Expose()
+  @IsNotEmpty()
+  @IsJWT()
+  public readonly token: string
+
+  @ApiProperty()
+  @Expose()
+  @IsNotEmpty()
+  @IsString()
+  public readonly password: string
+}
```

Comment:
naming suggestion - let's call this NewPasswordDto because this is what it contains